data/mapper/expr.go

```diff
@@ -21,20 +23,35 @@ func (mf *ExprMapperFactory) NewMapper(mappings map[string]interface{}) (Mapper,
 		return nil, nil
 	}
 
-	exprMappings := make(map[string]expression.Expr, len(mappings))
-
+	exprMappings := make(map[string]expression.Expr)
 	for key, value := range mappings {
-
-		if strVal, ok := value.(string); ok && len(strVal) > 0 && strVal[0] == '=' {
-			expr, err := mf.exprFactory.NewExpr(strVal[1:])
-			if err != nil {
-				return nil, err
+		if value != nil {
+			switch t := value.(type) {
+			case string:
+				if len(t) > 0 && t[0] == '=' {
+					expr, err := mf.exprFactory.NewExpr(t[1:])
+					if err != nil {
+						return nil, err
+					}
+					exprMappings[key] = expr
+				} else {
+					exprMappings[key] = expression.NewLiteralExpr(value)
+				}
+			//Object mapping
+			case map[string]interface{}, []interface{}:
+				objectExpr, err := NewObjectMapperFactory(mf.exprFactory).(*ObjectMapperFactory).NewObjectMapper(t)
```

Comment:
How do you distinguish between an object mapping and a object literal?

Reply:
@fm-tibco The logic, we don't need to distinguish literal or mapping.   All we going to object mapping, we resolve it if it is a mapping, otherwise, we do nothing.

Reply:
It should return an expression.NewLiteralExpr(value) if its not a mapping